codemagic.yaml

```diff
@@ -0,0 +1,13 @@
+workflows:
+    production:
+        name: Deploy to production
+        environment:
+            groups:
+                - aws-production
+                - production
+        scripts:
+            - name: Deploy to production
+              script: |
+                  set -e
+                  aws s3 sync ./your-first-flutter-app-to-appstore s3://labs.codemagic.io --delete --cache-control max-age=31536000,public
+                  aws cloudfront create-invalidation --distribution-id $DISTRIBUTION_ID --paths '/*'
```

Comment:
Consider invalidating only the paths that are actually present
```sh
INVALIDATE_PATHS=( $(aws s3 ls --recursive s3://${S3_BUCKET:-$DOMAIN_NAME} | awk '{print "/"$4}' | sed -e 's/index.html
aws cloudfront create-invalidation --distribution-id ${CLOUDFRONT_ID?} --paths ${INVALIDATE_PATHS[@]}
```

Reply:
in this case better to keep it simple